### DISCUSSION TITLE
PC died - Switched to Mac - ROR Curve weirdness

### DISCUSSION THREAD

**Discussion by DB-stoutheart** [artisan-roaster-scope/artisan#759] [Q&A]
My home built PC finally died, and now I'm on to my Mac for running Artisan.  I used to get REALLY nice ROR curves and now on my Mac they are very wavy, any ideas?  I do have a USB isolator inline that I had to use with the PC, not sure if thats the issue or not.

[SH1210_Nicaragua JINOTEGA JUAN DE DIOS CASTILLO_2021-12-15_1306.alog.pdf](https://github.com/artisan-roaster-scope/artisan/files/7722104/SH1210_Nicaragua.JINOTEGA.JUAN.DE.DIOS.CASTILLO_2021-12-15_1306.alog.pdf)

**Comment by CarefreeBuzzBuzz** [comment]
Have you compared your settings or did you reload your old ones.  A variety of noise factors could influence this.  I don’t see any HUGE problem with your graph.  You can adjust to that.  

Might help if you posted your various settings but this graph looks reasonable to me. 




Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team

**Reply by DB-stoutheart** [reply]
I imported the settings.  I'm a bit OCD, so I really liked seeing the nice smooth ROR is ideal.  Here is a previous one, not the same bean, but you get the idea.  I had no smoothing adjustments on this.

[SH1202_Guat Santa Cruz_2021-12-09_1445.pdf](https://github.com/artisan-roaster-scope/artisan/files/7722146/SH1202_Guat.Santa.Cruz_2021-12-09_1445.pdf)

**Comment by DB-stoutheart** [comment]
Here is a screenshot of my Curves-> Filter settings

![Screen Shot 2021-12-15 at 2 39 05 PM](https://user-images.githubusercontent.com/94983037/146253712-4e547592-3b0b-4b48-9dc4-785a96e69c8b.png)

**Comment by CarefreeBuzzBuzz** [comment]
Have you compared them with Polyfit and Optimal post roast turned off.  As
you suggested it could be the electrical noise without isolation on the
Mac.  If Artisan is running on both computers, its not Artisan.  Maybe even
the Phidget drivers are different for Mac and PC.  Can you use the isolator
with the Mac?

On Wed, Dec 15, 2021 at 12:39 PM DB-stoutheart ***@***.***>
wrote:

> Here is a screenshot of my Curves-> Filter settings
>
> [image: Screen Shot 2021-12-15 at 2 39 05 PM]
> <https://user-images.githubusercontent.com/94983037/146253712-4e547592-3b0b-4b48-9dc4-785a96e69c8b.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/759#discussioncomment-1816353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIW3J4B4T55VPS74AWVZ2SDURDVIZANCNFSM5KEOCLAQ>
> .
>


-- 

__________________
Michael Herbert, Head Roaster
*Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com>*
Artisan RoasterScope <https://artisan-scope.org/>Team

**Reply by DB-stoutheart** [reply]
Apologies but I'm not aware of either polyfit or the post roast settings, I'll look into those, unless you care to give me some guidance.  

The isolator is still being used with the Mac.

Appreciate your feedback.

**Comment by DB-stoutheart** [comment]
So I did a quick google search regarding Polyfit and Optimal Post Roast and nothing came up.  Do you happen to know where there is any documentation regarding this?

**Comment by CarefreeBuzzBuzz** [comment]
Always check the blog and quick start guide first.  

https://artisan-roasterscope.blogspot.com/2020/12/artisan-v244.html#more

https://artisan-scope.org/docs/curves/


----------------------------------------
Michael Herbert, Head Roaster
Order at www.carefreebuzzbuzz.com <http://www.carefreebuzzbuzz.com/>
Artisan RoasterScope  <https://artisan-scope.org/>Team

**Comment by bigl00z3** [comment]
Did you try reimporting the settings you had on the PC? If you didn't, save your current Mac settings and reimport your old PC settings. Then compare the differences in between your old and current settings,  mainly check the following parameters in the Curves settings:

- Delta Span (number of seconds)
- Smoothing (you currently are at 0, maybe you has smoothing before)
- As already mentioned by CarefreeBuzzBuzz, also check if the tick boxes were checked or not on the PC.

Nico

**Reply by DB-stoutheart** [reply]
Thanks Nico!  So I was able to access the information on my PC, and I can confirm both settings are exactly the same.  I'm going to try a roast today without the USB isolator and see if that makes a difference.

**Reply by bigl00z3** [reply]
Did you also check the Sampling Speed (/Config/Sampling...) in between the 2 computers' If it's different that could maybe explain.

**Comment by DB-stoutheart** [comment]
Well I've tried it all now, without adjusting the smoothing.  New cables, with isolator and without, adjust sampling speed and so on.  Not sure how to nail this down...if anyone has any additional recommendations let me know.  Its not a big deal throughout the beginning of the roast, its at the end that concerns me most.

**Comment by roasterdave** [comment]
You say you imported the settings from the PC to the Mac?  If so, then have you also moved the file _SH1210_Nicaragua SH1202_Guat Santa Cruz_2021-12-09_1445.alog_ to the Mac and opened it there in Artisan?  It should look identical to the way it does on the PC if the settings are the same.  If not then there is indeed a settings difference.  Post both the PC and the Mac settings files (menu Help>> Save Settings...) and we can compare them.  

PC and Mac both running Artisan v2.4.6?
